test/services/news/index.test.js

```diff
@@ -1,10 +1,71 @@
-'use strict';
+const { expect } = require('chai');
+const { ObjectId } = require('mongoose').Types;
+const sleep = require('util').promisify(setTimeout);
 
-const assert = require('assert');
 const app = require('../../../src/app');
+const News = require('../../../src/services/news/model').newsModel;
 
-describe('news service', function() {
-  it('registered the news service', () => {
-    assert.ok(app.service('news'));
-  });
+describe('news service', () => {
+	it('registers correctly', () => {
+		expect(app.service('news')).to.not.equal(undefined);
```

Comment:
what do you expect here instead? i would recommend to test not only not here.

Reply:
This is more or less a [pattern introduced by feathers](https://github.com/feathersjs/docs/blob/master/guides/chat/testing.md#testing-services)